### PR TITLE
Add index to `NewsItems.title` database column

### PR DESCRIPTION
**Pull request by anthotsang**
https://app.asana.com/0/683768070860536/881853263276627

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -306,6 +306,7 @@
     t.index ["coin_id"], name: "index_feed_sources_on_coin_id"
     t.index ["feed_type"], name: "index_feed_sources_on_feed_type"
     t.index ["feed_url"], name: "index_feed_sources_on_feed_url", unique: true
+    t.index ["id", "is_active", "feed_type"], name: "compound_index"
```
Comment (hanchang) [inline db/schema.rb]: Hm this looks new, what's the background?

File: db/schema.rb [hunk 2]
```diff
@@ -342,6 +343,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "market_caps", force: :cascade do |t|
```
Comment (hanchang) [inline db/schema.rb]: Let's stash these lines for this particular PR please.

File: db/schema.rb (hunk 1 above)
Comment (anthotsang) [reply db/schema.rb]: Looks like it's from a while ago but the schema.db was never updated.

https://github.com/coinfi/coinfi/blob/develop/db/migrate/20180928070531_add_is_active_index_to_feed_sources.rb

File: db/schema.rb (hunk 2 above)
Comment (anthotsang) [reply db/schema.rb]: Oops sorry, will clean that up.

File: db/schema.rb (hunk 1 above)
Comment (hanchang) [reply db/schema.rb]: Huh weird, the line below (310) should be the output for that migration though...

Let's go ahead and remove still unless I'm missing something?

Comment (anthotsang) [reply db/schema.rb]: Ah, you're right. I think this comes from a compound index I manually added when trying to optimize news items. :cry:

**Pull request merged**